imports/plugins/included/marketplace/client/templates/settings/sellerShopSettings.js

```diff
@@ -83,47 +80,39 @@ Template.sellerShopSettings.helpers({
   },
 
   currencyOptions() {
-    const instance = Template.instance();
+    const sellerShop = Reaction.getSellerShop();
 
-    if (instance.subscriptionsReady()) {
-      const sellerShop = Reaction.getSellerShop();
+    if (!sellerShop) {
+      return;
+    }
 
-      if (!sellerShop) {
-        return;
-      }
+    const currencies = [];
+    for (const currency in sellerShop.currencies) {
+      if ({}.hasOwnProperty.call(sellerShop.currencies, currency)) {
```

Comment:
We thought this was used enough times in RC to carry on with it. Happy to change it, but maybe we can setup a standard for it and change them all?
![image](https://cloud.githubusercontent.com/assets/670954/25057421/4ea30860-2178-11e7-9f67-f314bc256303.png)